app/models/journeyDomain/incident/equipment/EquipmentDomain.scala

```diff
@@ -64,18 +66,20 @@ object EquipmentDomain {
       case false => UserAnswersReader.apply(ItemNumbersDomain(Nil))
     }
 
-    lazy val sealsReadsByIncidentCode = IncidentCodePage(incidentIndex).reader.flatMap {
-      case x if x.code == "2" => sealsReads
-      case _                  => optionalSealsReads
+    lazy val sealsReadsByIncidentCode: Kleisli[EitherType, UserAnswers, SealsDomain] = IncidentCodePage(incidentIndex).reader.flatMap {
```

Comment:
`Kleisli[EitherType, UserAnswers, SealsDomain]` is the same as `UserAnswersReader[SealsDomain]`